### ISSUE TITLE
Unable to get control(accessibility) over addons/extensions in Firefox using Selenium

### ISSUE THREAD

**Issue by lokeshksymantec** [SeleniumHQ/selenium#518]
Steps to reproduce -
1)Install Firefox 36. 
2)﻿Search Videodownload helper and install the extension
3)
![screenshot 2015-05-05 17 12 41](https://cloud.githubusercontent.com/assets/9588987/7471923/4ec9b73e-f34a-11e4-9000-5e606cf3ab69.png)
Use selenium to capture controls for the installed extension

Actual Result : 
Unable to get control over addons/extensions in Firefox using Selenium 

Expected Result : 
Should get control over addons/extensions in Firefox using Selenium

**Comment by ddavison** [comment]
This is a question rather than an issue. Please send questions to the [selenium user group](https://groups.google.com/forum/#!forum/selenium-users)

For issues please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.

See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)
